### PR TITLE
update permission for template access

### PR DESCRIPTION
**Pull request by SANTOSH-MWP**
[https://mavenwave.atlassian.net/jira/software/c/projects/MC/boards/365?modal=detail&selectedIssue=MC-191&assignee=602279c2e612480069579d9d](url)

### CODE REVIEW

File: packages/backend/src/plugins/auth.ts [hunk 1]
```diff
@@ -21,29 +21,41 @@ export default async function createPlugin(
...
-
-            return ctx.issueToken({
-              claims: {
-                sub: userEntityRef,
-                ent: [userEntityRef],
-              },
-            });
-          },
+          resolver:
+            providers.microsoft.resolvers.emailMatchingUserEntityAnnotation(),
```
Comment (fosterm-mw) [inline packages/backend/src/plugins/auth.ts:24]: This broke the backend. I'm working on resolving these issues now.

**Review by fosterm-mw — CHANGES_REQUESTED** [review]

File: app-config.yaml [hunk 2]
```diff
@@ -18,10 +18,13 @@ app:
 organization:
   name: Maven Wave
 
+permission:
+  enabled: true
```
Comment (fosterm-mw) [inline app-config.yaml:22]: We need to do configuration in the auth backend to allow for this.

**Review by fosterm-mw — CHANGES_REQUESTED** [review]

File: app-config.yaml (hunk 2 above)
Comment (SANTOSH-MWP) [reply app-config.yaml:22]: that's right

**Review by fosterm-mw — APPROVED** [review]

**Review by SANTOSH-MWP — COMMENTED** [review]
changes looks good , the only piece index.ts need to verify which corresponds to this change.

**Pull request merged**